server/index.js

```diff
@@ -90,7 +99,7 @@ app.post("/v0/update", async (req, res) => {
 
 app.post("/v1/update", async (req, res) => {
   try {
-    res.send(await stores.addStoreData(req.body));
+    res.send(await stores.addStoreData(getFormDataWithUserIp(req)));
```

Comment:
since we haven't completed the move to the `v1` APIs, can we add this to `v0/update` as well?

Reply:
👍